common/app/model/content.scala

```diff
@@ -175,15 +175,15 @@ final case class Content(
 
   lazy val linkCounts = LinkTo.countLinks(fields.body) + fields.standfirst.map(LinkTo.countLinks).getOrElse(LinkCounts.None)
 
-  lazy val hasMultipleVideosInPage: Boolean = mainVideoCanonicalPath match {
-    case Some(_) => numberOfVideosInTheBody > 0
-    case None => numberOfVideosInTheBody > 1
-  }
+  lazy val mainMediaVideo = Jsoup.parseBodyFragment(fields.main).body.getElementsByClass("element-video")
 
-  lazy val mainVideoCanonicalPath: Option[String] = Jsoup.parseBodyFragment(fields.main).body.getElementsByClass("element-video").headOption.map { v =>
-    new URL(v.attr("data-canonical-url")).getPath.stripPrefix("/")
+  lazy val hasMultipleVideosInPage: Boolean = mainMediaVideo.isEmpty match {
```

Comment:
I don't understand why `mainVideoCanonicalPath` should not be an option. For an arbitrary piece of content, we don't know whether it has a main video at all, let alone whether that video has a canonical URL.